src/transformers/training_args.py

```diff
@@ -571,17 +557,20 @@ class TrainingArguments:
             Whether or not to compile the model using PyTorch 2.0
             [`torch.compile`](https://pytorch.org/get-started/pytorch-2.0/) (requires a nighlty install of PyTorch).
 
-            If set, the backend will default to `"inductor"` (can be customized with `torch_compile_backend`) and the
-            mode will default to `"default"` (can be customized with `torch_compile_mode`).
+            This will use the best defaults for the [`torch.compile`
+            API](https://pytorch.org/docs/2.0/generated/torch.compile.html?highlight=torch+compile#torch.compile). You
+            can customize the defaults with the argument `torch_compile_backend` and `torch_compile_mode` but we don't
+            guarantee any of them will work as the support is progressively rolled in in PyTorch.
+
+            This whole API is experimental and subject to changes in future releases.
```

Comment:
```suggestion
            This flag and the whole compile API is experimental and subject to change in future releases.
```

Reply:
apparently the idiom is "subject to change" (not changes) so changed it to that.